src/main/java/org/octopusden/releng/versions/NumericVersion.java

```diff
@@ -82,6 +82,14 @@ public int getBuildNumber() {
         return getItem(FIFTH_ITEM_INDEX);
     }
 
+    @Override
```

Comment:
Let's add more tests for getCounter in NumericVersionTest

Reply:
Added